packages/jaeger-ui/src/reducers/services.js

```diff
@@ -19,7 +19,7 @@ import { localeStringComparator } from '../utils/sort';
 
 const initialState = {
   // `services` initial value of `null` indicates they haven't yet been loaded
-  services: null,
+  services: [],
```

Comment:
how will we distinguish not loaded vs. empty?

Reply:
I was thinking on add a new flag on the state. This is some of the things I still need to add to this PR.